Cargo.toml

```diff
@@ -129,7 +129,7 @@ style_malloc_size_of = { package = "malloc_size_of", git = "https://github.com/s
 surfman = { git = "https://github.com/servo/surfman", rev = "c8d6b4b65aeab739ee7651602e29c8d58ceee123", features = ["chains"] }
 syn = { version = "2", default-features = false, features = ["clone-impls", "derive", "parsing"] }
 synstructure = "0.13"
-taffy = { version = "0.7", default-features = false, features = ["std", "serde", "grid"] }
+taffy = { path = "../taffy", default-features = false, features = ["std", "serde", "grid"] }
```

Comment:
This doesn't work.

Reply:
Yeah, didn't mean to commit this. I started this PR because I was wanting to upgrade taffy, but the latest version requires `1.83`...